pkg/neonvm/controllers/webhook.go

```diff
@@ -135,6 +137,16 @@ var _ webhook.CustomValidator = (*VMWebhook)(nil)
 // ValidateCreate implements webhook.CustomValidator
 func (w *VMMigrationWebhook) ValidateCreate(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
```

Comment:
Higher level question here: With #1300, the migration will already fail if the VM doesn't exist. Why do we need the same check in `ValidateCreate` ?

Separately, I'm not sure it's a good idea to perform k8s operations inside a webhook, because the webhook itself blocks other k8s operations. I think _in theory_ it should be fine, but that possibility of feedback makes me nervous.